LCM/dsc/engine/ca/CAInfrastructure/WebPullClient.c

```diff
@@ -1488,9 +1491,11 @@ MI_Result  IssueGetActionRequest( _In_z_ const MI_Char *configurationID,
     if (! (Strcasecmp( getActionStatus, GetActionResultOk) == 0 ||
         Strcasecmp( getActionStatus, GetActionResultGetConfiguration) == 0 ) )
     {
-    
+        MI_Char curlGetActionStatusCode[MAX_STATUSCODE_SIZE] = {0};
         *getActionStatusCode = GetDscActionCommandFailure;
-        r = GetCimMIError2Params(MI_RESULT_INVALID_PARAMETER, extendedError, ID_PULL_GETACTIONUNEXPECTEDRESULT, getActionStatus, url);
+        Stprintf(curlGetActionStatusCode, MAX_STATUSCODE_SIZE, MI_T("%d"), *getActionStatusCode);
+
+        r = GetCimMIError2Params(MI_RESULT_INVALID_PARAMETER, extendedError, ID_PULL_GETACTIONUNEXPECTEDRESULT, curlGetActionStatusCode, url);
```

Comment:
should this be logging getActionStatus instead? for this case we already know that getActionStatusCode is GetDscActionCommandFailure

Reply:
are we printing the getactionstatuscode anywhere though? I can switch it to getActionStatus, I just wasn't sure where we were using getActionStatusCode

Reply:
What i was trying to say is that this will always log GetDscActionCommandFailure... is that useful?

Also, the condition is testing against getActionStatus, so logging that instead may be more useful.

Reply:
Just updated.

Reply:
Also, I just finished another build where I tested curl_easy_strerror again, and I figured out what was going on - it is working as expected, so I added those back in as well

Reply:
I apologize for the last minute changes. I was waiting for my build to complete to confirm